queue_services/entity-digital-credentials/src/entity_digital_credentials/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.143.0'  # pylint: disable=invalid-name
+__version__ = '2.155.1'  # pylint: disable=invalid-name
```

Comment:
no more entity_digital_credentials

https://github.com/bcgov/lear/blob/main/queue_services/business-digital-credentials/pyproject.toml

Reply:
But there are code changes in the entity_digital_credentials folder ...?

Ref: https://github.com/bcgov/lear/pull/3730/files